src/commands/functions/functions-invoke.js

```diff
@@ -150,7 +150,9 @@ const functionsInvoke = async (nameArgument, options, command) => {
   const functions = await getFunctions(functionsDir)
   const functionToTrigger = await getNameFromArgs(functions, options, nameArgument)
 
-  let headers = {}
+  const headers = {
+    'user-agent': 'netlify-cli',
+  }
   let body = {}
 
   if (eventTriggeredFunctions.has(functionToTrigger)) {
```

Comment:
Do you think scheduled functions should fall on this branch?

Reply:
https://github.com/netlify/cli/pull/3689/commits/c61a2145c1a2f7e5575a9feb61d136a5ad70cba6#diff-9664db1fa0286aca02effcde71a026913114080a1c960e2257a357718ebb41eeR169

Reply:
I think this is addressed by https://github.com/netlify/cli/pull/3689/commits/c61a2145c1a2f7e5575a9feb61d136a5ad70cba6#diff-9664db1fa0286aca02effcde71a026913114080a1c960e2257a357718ebb41eeR165-R171